lib/gui/modules/analytics.js

```diff
@@ -141,21 +79,7 @@ analytics.service('AnalyticsService', function($log, $window, $mixpanel, Setting
    * });
    */
   this.logEvent = (message, data) => {
-    const debugData = utils.hideAbsolutePathsInObject(utils.makeFlatStartCaseObject(data));
-
-    if (SettingsModel.get('errorReporting') && isRunningInAsar()) {
```

Comment:
Again here, how do we ensure analytics are not sent anywhere if the user disabled error reporting?